clipper_admin/clipper_admin/docker/docker_container_manager.py

```diff
@@ -5,30 +5,35 @@
 import docker
 import logging
 import os
-import sys
-import random
 import time
-import json
 import tempfile
 from ..container_manager import (
     create_model_container_label, parse_model_container_label,
     ContainerManager, CLIPPER_DOCKER_LABEL, CLIPPER_MODEL_CONTAINER_LABEL,
     CLIPPER_QUERY_FRONTEND_CONTAINER_LABEL,
     CLIPPER_MGMT_FRONTEND_CONTAINER_LABEL, CLIPPER_INTERNAL_RPC_PORT,
-    CLIPPER_INTERNAL_QUERY_PORT, CLIPPER_INTERNAL_MANAGEMENT_PORT,
+    CLIPPER_INTERNAL_MANAGEMENT_PORT,
     CLIPPER_INTERNAL_METRIC_PORT, CLIPPER_INTERNAL_REDIS_PORT,
-    CLIPPER_DOCKER_PORT_LABELS, CLIPPER_METRIC_CONFIG_LABEL, ClusterAdapter)
-from ..exceptions import ClipperException
+    CLIPPER_DOCKER_PORT_LABELS, CLIPPER_METRIC_CONFIG_LABEL, ClusterAdapter,
+    CLIPPER_FLUENTD_CONFIG_LABEL)
 from requests.exceptions import ConnectionError
 from .docker_metric_utils import *
+from .logging.docker_logging_utils import (
+    get_logs_from_containers,
+    get_default_log_config
+)
+from clipper_admin.docker.logging.fluentd import Fluentd
 
 logger = logging.getLogger(__name__)
 
 
 class DockerContainerManager(ContainerManager):
+    # Logging-TODO Add SQLITE support
     def __init__(self,
                  cluster_name="default-cluster",
                  docker_ip_address="localhost",
+                 use_centralized_log=False,
+                 fluentd_port=24224,
```

Comment:
How about use `CLIPPER_INTERNAL_FLUENTD_PORT`? It is defined as 24224 in container_manager.py.

Reply:
That's a good idea. I will fix it!